packages/@aws-cdk/aws-apigateway/lib/model.ts

```diff
@@ -40,7 +51,133 @@ export class EmptyModel implements IModel {
  * }
  */
 export class ErrorModel implements IModel {
```

Comment:
Mark as @deprecated and point to `Model.ERROR_MODEL`

Reply:
Oups, done